edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/metrics/DeploymentMetrics.cs

```diff
@@ -191,5 +209,21 @@ Task UpdateCheckpointFile()
             File.WriteAllText(this.checkpointFile, this.systemTime.UtcNow.Ticks.ToString());
             return Task.CompletedTask;
         }
+
+        public void ReportManifestIntegrity(bool manifestCaPresent, bool integrtySectionPresent)
+        {
+            int manifestFlag = (manifestCaPresent || integrtySectionPresent) ? 1 : 0;
```

Comment:
Given the use case & interpretation of value, would it make sense to future proof this by turning into an `enum` instead of using an `int` here?

Reply:
To clarify what you're asking here - are you asking to make the `1` and `0` enum values, or the manifest flag itself an enum?

Reply:
I was thinking that `1` and `0` could be Enum type. Though I'm not entirely sure if the Prometheus sdk would automatically cast it back to int. (Thought I don't see why it wouldn't since it's the native behavior in C#).